docs/zkapps/testing-zkapps-lightnet.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Testing zkApps with Lightnet
+hide_title: true
+description: Use lightweight Mina blockchain network (Lightnet) for testing on a local blockchain before you test with a live network. 
+keywords:
+  - zkApp CLI
+  - lightnet
+  - zkApp testing
+  - Docker image
+  - using zk lightnet
+---
+
+# Testing zkApps with Lightnet
```

Comment:
This page repeats a lot of what's covered in lightnet.mdx. Is it necessary?

Reply:
I think you are right... we likely do not need a standalone explainer topic... I like your suggestion to keep all Lightnet info here in one place

Reply:
this is the accurate file and I've made the updates here